test/test_nn.py

```diff
@@ -1834,6 +1834,26 @@ def check():
         parameters.pop('p4')
         check()
 
+        # Check reverse works
+        forward = list(iter(parameter_dict))
+        backward = list(reversed(parameter_dict))
+        self.assertEqual(len(forward), len(backward))
+        n = len(forward)
+        for i in range(n):
+            self.assertIs(forward[i], backward[n - i - 1])
+        check()
+
+        # Check copy works
+        copy = parameter_dict.copy()
+        parameter_dict["p15"] = Parameter(torch.randn(10, 10))
+        copy["p16"] = Parameter(torch.randn(9, 10))
+
+        self.assertEqual("p15" in parameter_dict, True)
+        self.assertEqual("p15" in copy, False)
+        self.assertEqual("p16" in parameter_dict, False)
+        self.assertEqual("p16" in copy, True)
```

Comment:
I think you can just use assertTrue and assertFalse.
I would also go through and make sure all the other elements are the same between the old version and the copy.

Reply:
Thanks for pointing this out. I went ahead and pushed a change to address this